doc/modules/cassandra/pages/managing/operating/compaction/overview.adoc

```diff
@@ -0,0 +1,235 @@
+= Compaction overview
```

Comment:
why do the breadcrumbs work on the index.adoc but not this page or any of the other subpages ? 
<img width="566" alt="Screenshot 2023-12-08 at 20 33 02" src="https://github.com/apache/cassandra/assets/559444/7c4c66fa-5b5f-4ba9-b84b-b062cdd8512a">

Reply:
later